packages/vite/src/node/plugins/css.ts

```diff
@@ -901,23 +929,37 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
           collected.add(chunk)
 
           chunk.imports.forEach(collect)
```

Comment:
`chunk.imports` also should be appended to `css`.

Reply:
I changed the `collect` function to return `string`. Tried to clean the logic a bit.